.vscode/c_cpp_properties.json

```
@@ -0,0 +1,16 @@
+{
+  "configurations": [
+    {
+      "browse": {
+        "databaseFilename": "",
+        "limitSymbolsToIncludedHeaders": true
+      },
+      "includePath": [
+        "/opt/ros/noetic/include/**",
+        "/usr/include/**"
+      ],
+      "name": "ROS"
+    }
+  ],
+  "version": 4
+}
```

Calma, c_cpp_properties, vai ter C++ aqui?? huashuashuahsuash

---

Nunca se sabe kkkkkkk
~só commitei o que meu vscode colocou automaticamente, sla, nos outros repos tem kkkk~

---

Por mim daria pra tirar mesmo então, mas batata também, só tem que adicionar uma linha no final se for deixar ashuashushuahsuas.

```suggestion
{
  "configurations": [
    {
      "browse": {
        "databaseFilename": "",
        "limitSymbolsToIncludedHeaders": true
      },
      "includePath": [
        "/opt/ros/noetic/include/**",
        "/usr/include/**"
      ],
      "name": "ROS"
    }
  ],
  "version": 4
}

```